cumulusci/tasks/release_notes/generator.py

```diff
@@ -95,6 +103,83 @@ def _init_change_notes(self):
         return DirectoryChangeNotesProvider(self, self.directory)
 
 
+class ParentPullRequestNotesGenerator(BaseReleaseNotesGenerator):
+    """Aggregates notes from child pull requests in to a parent pull request
+    Attributes:
+        UNAGGREGATED_SECTION_HEADER: Str header for section on pull requests where
+        unaggregated chile pull requests are linked to.
+
+    Methods:
+        aggregate_child_change_notes: Aggregates change notes from child prs into parent
+        update_unaggregated_pr_header: Adds a new child pr to the section
```

Comment:
I'd leave the Attributes and Methods sections out of the class docstring and instead make sure the attribute has a comment next to it and the methods have their own docstrings.